llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -440,6 +442,10 @@ def construct_blocks(self, config: MPTConfig) -> nn.ModuleList:
             nn.ModuleList: The list of Transformer blocks.
         """
         block_args = self.extract_block_args(config.to_dict())
+        self.kv_cache_layers = None
+
+        if config.block_overrides is not None:
+            return self._construct_blocks_with_overrides(config, block_args)
 
         return nn.ModuleList([
```

Comment:
should probably make this block init a helper function so that you can call it inside the `construct_blocks_with_overrides`

Reply:
Done.